gradle/libs.versions.toml

```diff
@@ -4,6 +4,7 @@ grpc = "1.57.2"
 hypertrace-framework = "0.1.58"
 hypertrace-grpcutils = "0.12.2"
 hypertrace-kafka = "0.3.2"
+hypertrace-bom = "+"
```

Comment:
interesting?

Reply:
no change here except in structure. The catalog basically isn't declaring a bom version - the consumer can either set it or get latest. The plugin overrides this so if you ask for a specific catalog version, you get the matching bom version. If you ask for latest catalog, you get latest bom too.